src/Servers/IIS/IIS/test/Common.FunctionalTests/FrebTests.cs

```diff
@@ -56,6 +54,9 @@ public async Task CheckCommonFrebEvents()
 
         await result.HttpClient.GetAsync("HelloWorld");
 
+        // Dispose the client to speed up tests
+        result.HttpClient.Dispose();
```

Comment:
Should we be doing this in all of our similar tests?  I can do that in a follow-up PR if this is noticeable